src/styles/globals.scss

```diff
@@ -1,6 +1,15 @@
 @import './variables';
-@import "~bulma";
 @import './skeleton';
+@import "~bulma";
```

Comment:
Tanken ved å ha bulma importen før skeleton, var for å kunne overkøyre noko av skeleton stylinga som bulma kanskje hadde lagt inn